src/mcp/server/fastmcp/server.py

```diff
@@ -68,7 +68,7 @@ class Settings(BaseSettings, Generic[LifespanResultT]):
 
     # Server settings
     debug: bool = False
-    log_level: Literal["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"] = "ERROR"
+    log_level: Literal["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"] = "INFO"
```

Comment:
@dsp-ant I saw you accepted a PR changing this, but I think `INFO` is a better default. 🤔